main/io.home-assistant/manifest.tt

```diff
@@ -7,26 +7,25 @@ class @io.home-assistant
 #[license_gplcompatible=true]
 #[subcategory="home"]
 #[website="https://home-assistant.io"]
-#[child_types=["org.thingpedia.iot.light-bulb",
-               "org.thingpedia.iot.switch"/*
-               "io.home-assistant.climate",
+#[child_types=["org.thingpedia.iot.battery", "org.thingpedia.iot.humidity", "org.thingpedia.iot.illuminance", "org.thingpedia.iot.light-bulb", "org.thingpedia.iot.motion", "org.thingpedia.iot.switch", "org.thingpedia.iot.temperature", "org.thingpedia.iot.uv" ]]
```

Comment:
If you don't mind, keep one element per line, it's a bit more readable.

Reply:
ok